streambed-machine/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "streambed-machine"
```

Comment:
I think we can move `streambed-machine` to streambed once the other crates are published.